pom.xml

```diff
@@ -79,22 +79,22 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>scm-api</artifactId>
-            <version>1.3</version>
+            <version>1.4-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>branch-api</artifactId>
-            <version>1.11</version>
+            <version>1.12-SNAPSHOT</version>
```

Comment:
This is the version that depends on scm-api 1.4-SNAPSHOT, I want to keep dependencies aligned